src/ResourceManager/ServiceBus/Commands.ServiceBus/Cmdlets/Topic/GetAzureServiceBusTopicAuthorizationRules.cs

```diff
@@ -1,80 +0,0 @@
-﻿// ----------------------------------------------------------------------------------
-//
```

Comment:
Why are you deleting these cmdlets?

Reply:
Authorizationrule cmdlets are generic under ServicebusAuthorizationrule. need to check how this file removal came in the PR.

Reply:
Please let me know when this is resolved.

Reply:
Approved offline.